sdk/communication/AzureCommunication/Source/Authentication/CommunicationTokenCredential.swift

```diff
@@ -55,7 +55,7 @@ public typealias TokenRefreshOnCompletion = (String?, Error?) -> Void
         - option: Options for how the token will be refreshed
      - Throws: `AzureError` if the provided token is not a valid user token.
      */
-    public init(with option: CommunicationTokenRefreshOptions) throws {
+    public init(withOptions option: CommunicationTokenRefreshOptions) throws {
```

Comment:
nit: should 'option:' be renamed to 'options;'?

as it is withOptions, and CommunicationTokenRefreshOptions, both have 's'